src/image/ImageAnnotator.ts

```diff
@@ -42,6 +42,9 @@ export default class ImageAnnotator extends BaseAnnotator {
     }
 
     getManagers(parentEl: HTMLElement, referenceEl: HTMLElement): Set<Manager> {
+        const isCurrentFileVersion = getIsCurrentFileVersion(this.store.getState());
```

Comment:
Does this all work correctly if these store values change outside of the `getManagers` flow? Should we introduce something like the BaseManager `style` method that gets called when these values change in the store?

Reply:
I think that's unnecessary at this point. If the `fileid` or `iscurrent` properties change, a new preview is triggered and annotations is reinstantiated. If at a later point we introduce a field that can change on the fly here then we'd need to solve that problem